certified-connectors/Conversica/README.md

```diff
@@ -0,0 +1,66 @@
+# Conversica API Connector
+
```

Comment:
Hi @edgarcastelo, 

Please update readme.md file as per the readme template in this link. Please make sure that you added all the required sections. https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md